app/models/claims/sort.rb

```diff
@@ -37,34 +37,44 @@ def sort(column, direction)
   #
 
   def nulls_at_top_asc(direction)
```

Comment:
not your name but perhaps this should be name sort_nulls or nulls_direction.  kind of like the NULLS text being in the method too
```
def sort_nulls(direction)
  'NULLS ' + (direction == 'asc' ? 'FIRST' : 'LAST')
end
```